aws_lambda_powertools/utilities/parser/models/apigw.py

```diff
@@ -68,6 +68,13 @@ class APIGatewayEventRequestContext(BaseModel):
     routeKey: Optional[str]
     operationName: Optional[str]
 
+    @root_validator()
+    def check_message_id(cls, values):
```

Comment:
you cant define a root validator like that . when you use @root_validator it needs to be inside the class that uses it. Usually you define it outside when you want to reuse it at several places but then you define it differetly inside the classes which use it.

Reply:
@risenberg-cyberark - ok, interesting. Coz this passes the unit tests now. And before it was not actually catching invalid events.

Reply:
https://pydantic-docs.helpmanual.io/usage/validators/#root-validators
that's very odd, i'm using the documented approach of root validators

Reply:
it's because it has an extra () by mistake in the decoator

Reply:
can you remove the () from the decorator?

Reply:
@risenberg-cyberark  - did this, but this does not change much